### DISCUSSION TITLE
Carlinkit not detected / Android not loading

### DISCUSSION THREAD

**Discussion by BroRidersMTL** [tesla-android/issue-tracker#88] [General]
Hi guys, finally got around to switch to single board and use one for each vehicle but im having the following issues

Pi4 #1 - won't detect the carlinkit but its powered and available on my phone bluetooth & wifi

Pi4 #2 - it seems not getting to android i have erased and reinstalled 3 times now and always the same issue (attached video)

Any help would be appreciated

https://user-images.githubusercontent.com/105889119/193104579-ca77b710-af36-4ab1-8fef-384b0de88888.MOV

**Comment by BroRidersMTL** [comment]
Pi4 #2 - Issue resolved bad hdmi-cs2 card

**Reply by RetoHB** [reply]
Probably same issue here. But before buying a new card I will wait for the next release.

**Comment by BroRidersMTL** [comment]
Pi4 #1 - Issue resolved plug ethernet to pi4 and update autokit
